components/about/OurTeam.vue

```diff
@@ -149,8 +149,8 @@ export default {
           designation: 'DevOps',
           image: 'Lakshmikanth',
           links: {
-            linkedin: 'linkedin.com/in/lakshmikanth-bharathi-2a4754b8',
-            twitter: 'https://twitter.com/LakshmikanthBh8',
+            linkedin: 'http://linkedin.com/in/lakshmikanth-bharathi-2a4754b8',
```

Comment:
[nit] Use `https` for consistency.

Reply:
done, [commit link](https://github.com/arcana-network/website/pull/84/commits/2b91f87f700cb4e6a66adae544b0785d77b361e0)